src/main/java/shortestpath/ShortestPathConfig.java

```diff
@@ -381,18 +381,29 @@ default boolean showTransportInfo() {
         return true;
     }
 
+    @ConfigItem(
+            keyName = "scrollFairyRingList",
+            name = "Scroll Fairy Ring List",
+            description = "Whether the list of fairy ring options is scrolled to the one needed for the shortest path",
+            position = 31,
+            section = sectionSettings
+    )
+    default boolean scrollFairyRingList() {
```

Comment:
I don't think a config option for this is necessary. Always scrolling to the fairy ring code is fine.